layouts/partials/navbar.html

```diff
@@ -1,5 +1,5 @@
 {{ $homeLink := .Site.Params.Homepage }}
-{{ if .IsHome }}
```

Comment:
This is used for Landing Page (/), You may test it here https://eternity.bora.sh/  
The reason is, when you are in Landing Page, We wait you to click "enter" button (that's why Logo doesn't go to .Homepage). All other pages (non "/") will use .Site.Params.Homepage  
I think it makes sense